Adapt `Animation` events to new structure

teoli2003 opened the pull request:
This PR updates the `Animation` interface for new-style event pages. (see the [template](https://developer.mozilla.org/en-US/docs/MDN/Structures/Page_types/API_event_subpage_template))

The `on-*` event handler pages are redirected to the `eventname_event` pages.

`yarn content move Web/API/Animation/oncancel Web/API/Animation/cancel_event`
`yarn content move Web/API/Animation/onfinish Web/API/Animation/finish_event`
`yarn content move Web/API/Animation/onremove Web/API/Animation/remove_event`

BCD: mdn/browser-compat-data#14914

teoli2003 committed:
Adapt Animation events to new structure

teoli2003 committed:
Fix a few flaws.

teoli2003 committed:
Fix erroneous tags (bork sidebar)

Elchi3 commented on files/en-us/web/api/animation/cancel_event/index.md:
````diff
@@ -0,0 +1,64 @@
+oncancel = event => { }
+```
+
+### Value
````
This "Value" section is not in the template. I don't think it is useful.

teoli2003 replied:
Sorry forgot this one.

Elchi3 commented on files/en-us/web/api/animation/remove_event/index.md:
````diff
@@ -1,32 +1,39 @@
-animation.onremove = removeHandler;
+```js
+addEventListener('remove', event => { })
+remove = event => { }
````
```suggestion
onremove = event => { }
```

Elchi3 commented on files/en-us/web/api/animation/remove_event/index.md:
```diff
@@ -1,32 +1,39 @@
+{{ APIRef("Web Animations") }}
 
-The {{domxref("Animation")}} interface's **`onremove`** property (from the [Web Animations API](/en-US/docs/Web/API/Web_Animations_API)) is the event handler for the {{event("remove")}} event. This event is sent when the animation is removed (i.e., put into an `active` replace state).
+The **`remove`** event of the {{domxref("Animation")}} interface is fore when the animation is removed (i.e., put into an `active` replace state).
```
```suggestion
The **`remove`** event of the {{domxref("Animation")}} interface fires when the animation is removed (i.e., put into an `active` replace state).
```

Elchi3 commented on files/en-us/web/api/animation/cancel_event/index.md:
```diff
@@ -0,0 +1,64 @@
+
+- [Web Animations API](/en-US/docs/Web/API/Web_Animations_API)
+- {{domxref("Animation")}}
+- The {{event("cancel")}} event
```
```suggestion
```

Elchi3 reviewed (changes requested):
Thanks, some change requests but looks good overall.

Aside: I see a lot of whitespace changes... I thought we updated many files recently to fix these things. Is something wrong with your editor settings or has mdn/content's editorconfig changed or something?

teoli2003 committed:
fix missing on

Co-authored-by: Florian Scholz <fs@florianscholz.com>

teoli2003 committed:
Grammar fix

Co-authored-by: Florian Scholz <fs@florianscholz.com>

teoli2003 committed:
Remove link

Co-authored-by: Florian Scholz <fs@florianscholz.com>

teoli2003 committed:
Remove value section

teoli2003 commented:
> Thanks, some change requests but looks good overall.
> 
> Aside: I see a lot of whitespace changes... I thought we updated many files recently to fix these things. Is something wrong with your editor settings or has mdn/content's editorconfig changed or something?

I modified them: they were unicode non-breakable spaces and I just replaced them with regular spaces. Some editors put them all over the place when you create new pages. (I think mainly Microsoft Word).
